habitat_baselines/rl/ppo/policy.py

```diff
@@ -171,6 +208,13 @@ def all_policy_tensors(self):
         for c in self.policy_components:
             yield from c.buffers()
 
+    def aux_loss_parameters(self):
+        res = {}
+        for k, v in self.aux_loss_modules.items():
+            res[k] = v.parameters()
+
+        return res
```

Comment:
```suggestion
        return {k: v.parameters() for k, v in self.aux_loss_modules.items()}
```
faster and one liner.